lib/Bindings/Python/circt/support.py

```diff
@@ -348,6 +348,10 @@ def __getattr__(self, name):
       value = self.opview.results[index]
       return OpOperand(self.opview.operation, index, value, self)
 
+    # Forward "attributes" attribute from the operation.
```

Comment:
I think this makes sense, since these things could be on any op. This will make it easier to access attributes in general, and avoid needing specific support at the Python level for everything that has SVAttributes.

Reply:
It doesn’t seem to me that this is strictly related to this PR. Nonetheless, I agree.

Reply:
Committed by https://github.com/llvm/circt/commit/ff70be1db9d0be39e3abf34484e6715d901a7373